### PR TITLE
Upgrade to Gradle experimental plugin 0.6.0-alpha5

### PR DESCRIPTION
**Pull request by smarek**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -50,6 +52,9 @@ model {
         create("all") {
         }
     }
+    android.lintOptions {
```
Comment (M66B) [inline app/build.gradle]: Lint will help prevent errors, so why turn it off?

Comment (smarek) [inline app/build.gradle]: Lint rules got more strict, with options turned off, you can see those errors, but won't prevent you from building the application. I was able to run the app with previous gradle version (2.9), with 2.10 it failed due to Lint results.

Comment (M66B) [inline app/build.gradle]: Good reason, but we should strive to solve these warnings/errors.

**Comment by M66B** [comment]
I am in the middle of rewriting/testing a better TCP handling, after I have finished that, I will merge this.

Thanks for creating this pull request.

File: app/build.gradle (hunk 1 above)
Comment (smarek) [inline app/build.gradle]: No doubts about that, currently there are few errors, specifically `NewApi` (SinkholeService.java:268) and `StrictFormatInvalid` (ActivityForward.java:63), others are *just warnings*

**Comment by licaon-kter** [comment]
AS2 entered beta already, maybe you can look at it later too: https://github.com/M66B/NetGuard/issues/211 ;)

**Comment by M66B** [comment]
In my experience beta products often cost you time, which I rather spent in developing NetGuard.
As soon as 2.0 stable is released, I will upgrade.

**Comment by M66B** [comment]
The rewritten TCP stack seems to work okay now.
The TCP close handling has been revise and receive window calculation has been implemented.

**Pull request merged**